### DISCUSSION TITLE
val_check_interval every N global steps?

### DISCUSSION THREAD

**Discussion by senzeyu** [Lightning-AI/pytorch-lightning#11152] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hello, is there a way to call validation for every N global steps? For example, we could have vall_check_interval greater than the number of batches in the training dataset.

**Comment by carmocca** [comment, marked as answer]
Unfortunately, not at the moment.

There's an issue for it here https://github.com/PyTorchLightning/pytorch-lightning/issues/8135
